src/main/webapp/app/apollon-diagrams/apollon-diagram-tutor.component.ts

```diff
@@ -354,4 +367,32 @@ export class ApollonDiagramTutorComponent implements OnInit, OnDestroy {
     previousState() {
         this.router.navigate(['course', this.modelingExercise.course.id, 'exercise', this.modelingExercise.id, 'assessment']);
     }
+
+    private highlightElementsWithConflict() {
+        const state: State = this.apollonEditor.getState();
+        const entitiesToHighlight: string[] = state.entities.allIds.filter((id: string) => {
+            if (this.conflicts.has(id)) {
+                return true;
+            }
+            if (state.entities.byId[id].attributes.find(attribute => this.conflicts.has(attribute.id))) {
+                console.log('attribute found');
```

Comment:
please remove all console.log statements for production

Reply:
hups